mobile/lib/modules/asset_viewer/services/image_viewer.service.dart

```diff
@@ -39,7 +39,8 @@ class ImageViewerService {
           final failedResponse =
               imageResponse.statusCode != 200 ? imageResponse : motionReponse;
           _log.severe(
-            "Motion asset download failed with status - ${failedResponse.statusCode} and response - ${failedResponse.body}",
+            "Motion asset download failed",
+            "Status: ${failedResponse.statusCode}\nResponse: ${failedResponse.body}",
```

Comment:
Can the HTTP errors be refactored to a custom object for errors and the actual logging string be moved to the `toString` method of the custom class instead?

Reply:
Makes sense. I'll probably need a few days though because a) I'm new to Dart and b) family keeps me busy 😉

Reply:
Sorry, still didn't get around to do this. I hope tomorrow or on the weekend.

Reply:
I went a slightly different route by adding an extension for the `Response` class. So whenever there's something to log for an HTTP call, the `toLoggerString()` method can be called. I hope that is fine as well. If not, I can add a wrapper class with a `toString()` instead, but I don't see advantages with that.